lib/minesweeper.rb

```diff
@@ -0,0 +1,55 @@
+require_relative 'minefield'
+require_relative 'turn'
+
+class Minesweeper
+  def initialize
+    @mines = Minefield.new
+    intro
+    loop do
+      print_game_board
+      take_a_turn
+      check_for_game_end
+    end
+  end
+
+  private
+
+  def intro
+    print "welcome to\nMINESWEEPER\n\nCOMMANDS:\n'C' to clear\n'A' to auto-clear\n'F' to flag\n'U' to un-flag\nThe ⚑ shows you how many flags are left"
```

Comment:
I would separate this into multiple print/puts statements for readability, especially since you have newlines